frontend/src/components/GoalsTable/__tests__/GoalsTable.js

```diff
@@ -391,8 +391,8 @@ describe('Goals Table', () => {
       await screen.findByText('TTA goals and objectives');
 
       expect(screen.getAllByRole('cell')[0]).toHaveTextContent(/needs status/i);
-
-      expect(document.querySelector('.fa-exclamation-circle')).toHaveAttribute(
+      screen.debug(undefined, 100000);
```

Comment:
```suggestion
```

Reply:
I removed the debug. 

It seems the current Cicle CI env var is for "Head-Start-TTADP" project.

Reply:
The new FA has also been deployed to DEV works as expected.